### DISCUSSION TITLE
[Bug]: It's not related to YimMenu but i dont know wherelse to ask this...

### DISCUSSION THREAD

**Discussion by malloc2k** [YimMenu/YimMenu#2815] [Q&A]
### Describe the bug

I wanted to reverse engineer GTAV using something like IDA or x64DGB but I have no clue on how to go...

I have did this with CSGO and CS2 but with those you just find stuff on client.dll and I dont know how to do the similar in GTAV, 
can any good soul help me start

### Steps To Reproduce

i dont know 

### Expected Behavior

...

### Logs and/or screenshots

```
<paste your logs here>
```

### Store

Rockstar

### Language

English

### YimMenu Language

English (US)

### Additional context

Thanks :(

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by Yimura** [comment]
Simple answer: load the game up, dump it from memory and analyse it in IDA/Ghydra

Long answer: the exact steps are on UnknownCheats

**Reply by malloc2k** [reply]
Where can I find this? Can you give me a link? 

Thanks
